cmd/soroban-rpc/internal/test/metrics_test.go

```diff
@@ -28,5 +22,14 @@ func TestMetrics(t *testing.T) {
 		runtime.Version(),
 		config.Version,
 	)
-	require.Contains(t, responseString, buildMetric)
+	require.Contains(t, metrics, buildMetric)
+}
+
+func getMetrics(test *Test) string {
+	response, err := http.Get(test.adminURL() + "/metrics")
```

Comment:
this is probably ok here, but just fyi, this is usually being done using "path.Join", which would eliminate double `/`, etc.

Reply:
sorry - I should have written [`url.JoinPath`](https://pkg.go.dev/net/url@go1.19beta1#JoinPath)